developer_manual/digging_deeper/search.rst

```diff
@@ -183,7 +183,7 @@ Next, you'll see a dummy provider that returns a static set of results using the
         }
 
         public function search(IUser $user, ISearchQuery $query): SearchResult {
-            return MySearchResultEntry::complete(
```

Comment:
in that case the definition of `MySearchResultEntry` is obsolete

Reply:
yeah that class should be wiped completly

Reply:
the thing is that in your example `MySearchResultEntry` extends `SearchResultEntry` that do not contains the method `complete` (from `SearchResult`)

Reply:
Right, because that is documented incorrectly. Of course the search method should use `\OCP\Search\SearchResult::complete` to build the result